matter_demos.xml

```diff
@@ -1,723 +1,1153 @@
-    <property key="demos.blurb" value="Matter - Thermostat App"/>
-    <property key="core.partCompatibility" value=".*"/>
-    <property key="core.boardCompatibility" value="brd4187c"/>
-    <property key="demos.imageFile" value="Examples/thermostat-app-wifi/rs9116/BRD4187C/chip-efr32-thermostat-example.s37"/>
-    <property key="core.readmeFiles" value="https://github.com/SiliconLabs/matter#readme"/>
-    <property key="core.quality" value="PRODUCTION"/>
-    <property key="filters" value="Wireless\ Technology|Matter"/>    
-    <description>This is a Matter Thermostat Application demo for BRD4187C to be used with RS9116 Wi-Fi expansion board</description>
-   </demo> 
-</demos>
\ No newline at end of file
+	<demo name="brd4187c.demo.window_app_wifi_wf200" label="Matter - Window over Wi-Fi">
+		<property key="demos.blurb" value="Matter - Window app"/>
+		<property key="core.partCompatibility" value=".*"/>
+		<property key="core.boardCompatibility" value="brd4187c"/>
+		<property key="demos.imageFile" value="Examples/window-app-wifi/wf200/BRD4187C/chip-efr32-window-example.s37"/>
+		<property key="core.readmeFiles" value="https://github.com/SiliconLabs/matter#readme"/>
+		<property key="filters" value="Wireless\ Technology|Matter"/>
+		<property key="core.quality" value="PRODUCTION"/>
+		<description>This is a Matter Window Application for BRD4187C to be used with WF200 Wi-Fi expansion board</description>
+	</demo>
+	<demo name="brd4161a.demo.window_app_wifi_wf200" label="Matter - Window over Wi-Fi">
+		<property key="demos.blurb" value="Matter - Window app"/>
+		<property key="core.partCompatibility" value=".*"/>
+		<property key="core.boardCompatibility" value="brd4161a"/>
+		<property key="demos.imageFile" value="Examples/window-app-wifi/wf200/BRD4161A/chip-efr32-window-example.s37"/>
+		<property key="core.readmeFiles" value="https://github.com/SiliconLabs/matter#readme"/>
+		<property key="filters" value="Wireless\ Technology|Matter"/>
+		<property key="core.quality" value="PRODUCTION"/>
+		<description>This is a Matter Window Application for BRD4161A to be used with WF200 Wi-Fi expansion board</description>
+	</demo>
+	<demo name="brd4186c.demo.window_app_wifi_wf200" label="Matter - Window over Wi-Fi">
+		<property key="demos.blurb" value="Matter - Window app"/>
+		<property key="core.partCompatibility" value=".*"/>
+		<property key="core.boardCompatibility" value="brd4186c"/>
+		<property key="demos.imageFile" value="Examples/window-app-wifi/wf200/BRD4186C/chip-efr32-window-example.s37"/>
+		<property key="core.readmeFiles" value="https://github.com/SiliconLabs/matter#readme"/>
+		<property key="filters" value="Wireless\ Technology|Matter"/>
+		<property key="core.quality" value="PRODUCTION"/>
+		<description>This is a Matter Window Application for BRD4186C to be used with WF200 Wi-Fi expansion board</description>
+	</demo>
+	<demo name="brd4161a.demo.window_app_wifi_rs9116" label="Matter - Window over Wi-Fi">
```

Comment:
Two binaries are missing:
1. window_app_wifi_rs9116 for BRD4186C
2. window_app_wifi_rs9116 for BRD4187C